packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/NuxtSetupSteps.tsx

```diff
@@ -0,0 +1,162 @@
+          </pre>
+        </Flex>
+      </Flex>
+    </StepSection>
+    <StepSection
+      stepNumber={2}
+      title={"Update your Nuxt config"}
+      isOpen={activeStep === 2}
+      onOpenStep={onOpenStep(2)}
+    >
+      <Flex sx={{ flexDirection: "column" }}>
+        <Text sx={{ color: "textClear", mb: 2 }}>
+          In your nuxt.config.js file, you need to add at the beginning the
+          following line:
+        </Text>
+        <Flex sx={{ mb: 2 }}>
+          <pre
+            style={{ overflowX: "auto", padding: 8, border: "solid 1px grey" }}
+          >
+            import smConfig from "./sm.json"
+          </pre>
+        </Flex>
+        <Text sx={{ color: "textClear", mb: 2 }}>
+          In your nuxt.config.js file, you need to add at the beginning the
+          following line:
+        </Text>
+        <Flex>
+          <pre
+            style={{ overflowX: "auto", padding: 8, border: "solid 1px grey" }}
+          >
+            {NuxtConfigInstructions}
+          </pre>
+        </Flex>
+      </Flex>
+    </StepSection>
+    <StepSection
+      stepNumber={3}
+      title={"Create a page for Slice Canvas"}
+      isOpen={activeStep === 3}
+      onOpenStep={onOpenStep(3)}
+    >
+      <Flex sx={{ flexDirection: "column" }}>
+        <Text sx={{ color: "textClear", mb: 2 }}>
+          In your “pages” directory, create a file called _canvas.jsx and add
+          the following code. This page is the route you hit to preview and
+          develop your components.
+        </Text>
+        <Flex>
+          <pre
+            style={{ overflowX: "auto", padding: 8, border: "solid 1px grey" }}
+          >
+            {SliceCanvasPageCreationInstruction}
+          </pre>
+        </Flex>
+      </Flex>
+    </StepSection>
+    <StepSection
+      stepNumber={4}
+      title={"Update sm.json"}
+      isOpen={activeStep === 4}
+      onOpenStep={onOpenStep(4)}
+    >
+      <Flex sx={{ flexDirection: "column" }}>
+        <Text sx={{ color: "textClear", mb: 2 }}>
+          Update your <Text variant={"pre"}>sm.json</Text> file with the
+          property <Text variant={"pre"}>localSliceCanvasURL</Text> in the shape
+          of <Text variant={"pre"}>http://localhost:PORT/PATH</Text>.
+        </Text>
+        <Flex>
+          <pre
+            style={{ overflowX: "auto", padding: 8, border: "solid 1px grey" }}
+          >
+            {"// eg:\n" +
+              '"localSliceCanvasURL": "http://localhost:3000/_canvas"'}
+          </pre>
+        </Flex>
+      </Flex>
+    </StepSection>
+    <StepSection
+      stepNumber={5}
+      title={"Check configuration"}
+      isOpen={activeStep === 5}
+      onOpenStep={onOpenStep(5)}
+    >
+      <Flex>
+        <Text sx={{ color: "textClear" }}>
+          After you’ve done the previous steps, we need to check that everything
+          works in order.
+        </Text>
+      </Flex>
+    </StepSection>
+  </>
+);
+
+const SliceCanvasPageCreationInstruction =
```

Comment:
you know you can use template strings for this? (saves typing `\n` and +  so much)

Reply:
Good catch :)